vnext/JSI/Desktop/ChakraJsiRuntime_core.cpp

```diff
@@ -14,6 +14,27 @@
 
 namespace Microsoft::JSI {
 
+namespace {
+
+#if !defined(CHAKRACORE_UWP)
+// This is very wierd. This should match with the definition of VS_VERSIONINFO
```

Comment:
>wierd [](start = 16, length = 5)

(nit) misspelled

Reply:
Since this PR is already merged, I fixed this. Will check-in as part of a later PR.

---
In reply to: [333109439](https://github.com/microsoft/react-native-windows/pull/3366#discussion_r333109439) [](ancestors = 333109439)